Instagram_App/Instagram_App/ViewControllers/search/ModelCollectionViewCell.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ModelCollectionViewCell.swift
+//  Instagram_App
+//
+//  Created by coder on 23.10.2022.
+//
+
+import Foundation
+
+/// структура для передачи данных в ячейку истории
+struct CollectionCell {
+    var nameImage: [String]
```

Comment:
```suggestion
    let nameImage: String
```
во всех моделях убрать массивы